blog/config/nav.yml

```diff
@@ -6,7 +6,7 @@ nav:
     - Eventing: /docs/eventing/
     - Code samples: /docs/samples/
     - Reference: /docs/reference/
-    - "Join the Community ➠": /docs/community/
+    - "Community ➠": /docs/community/
```

Comment:
Do we want to take out the arrow? I _think_ that was being used for links that redirected to Hugo pages (we took it out for the blog when it moved to MK Docs).